src/models/tools/geometry/geometry-content.ts

```diff
@@ -60,7 +64,8 @@ export const GeometryContentModel = types
         destroyBoard,
         resizeBoard,
         addPoint,
-        applyChange: _applyChange
+        applyChange: _applyChange,
+        syncChange
```

Comment:
Could you talk a little bit about the difference between applying and synchronizing changes? It looks like synced changes aren't kept in a list - is this to save space over time?

Reply:
All changes are kept in the array. The component that initiates a change uses `applyChange` to apply the change to the board and add it to the array. Components that are notified of the change use `syncChanges` to apply the change to their board instance, but they don't add it to the array because it's already there -- the model is shared.

Reply:
Ahh got it, and that also explains why the tool keeps a separate count of changes